packages/wonder-blocks-announcer/src/__tests__/util/dom.test.ts

```diff
@@ -0,0 +1,133 @@
+    describe("createRegionWrapper", () => {
+        test("it creates a polite region wrapper element", () => {
+            const element = createRegionWrapper("polite");
+
+            expect(element.tagName).toBe("DIV");
+            expect(element.id).toEqual("wbAWrap-polite");
+        });
+
+        test("it creates an assertive region wrapper element", () => {
+            const element = createRegionWrapper("assertive");
+
+            expect(element.tagName).toBe("DIV");
+            expect(element.id).toEqual("wbAWrap-assertive");
+        });
+    });
+
+    describe("createDuplicateRegions", () => {
+        test("it creates multiple polite region elements", () => {
+            const wrapper = document.createElement("div");
+            const dictionary = new Map();
+
+            const regionList = createDuplicateRegions(
+                wrapper,
+                "polite",
+                2,
+                dictionary,
+            );
+
+            expect(regionList.length).toBe(2);
+            expect(regionList[0].id).toBe("wbARegion-polite0");
+            expect(regionList[1].id).toBe("wbARegion-polite1");
+            expect(dictionary.size).toBe(2);
+        });
+
+        test("it creates multiple assertive region elements", () => {
+            const wrapper = document.createElement("div");
+            const dictionary = new Map();
+
+            const regionList = createDuplicateRegions(
+                wrapper,
+                "assertive",
+                2,
+                dictionary,
+            );
+
+            expect(regionList.length).toBe(2);
+            expect(regionList[0].id).toBe("wbARegion-assertive0");
+            expect(regionList[1].id).toBe("wbARegion-assertive1");
+            expect(dictionary.size).toBe(2);
+        });
+    });
+
+    describe("createRegion", () => {
+        test("it creates a polite Live Region element", () => {
+            const dictionary = new Map();
+            const region = createRegion("polite", 0, dictionary);
+
+            expect(region.id).toBe("wbARegion-polite0");
+            expect(region.getAttribute("aria-live")).toBe("polite");
+            expect(region.getAttribute("role")).toBe("log");
+            expect(dictionary.size).toBe(1);
+        });
+
+        test("it creates an assertive Live Region element", () => {
+            const dictionary = new Map();
+            const region = createRegion("assertive", 0, dictionary);
+
+            expect(region.id).toBe("wbARegion-assertive0");
+            expect(region.getAttribute("aria-live")).toBe("assertive");
+            expect(region.getAttribute("role")).toBe("log");
+            expect(dictionary.size).toBe(1);
+        });
+
+        test("it allows the role to be overridden", () => {
+            const dictionary = new Map();
+            const region = createRegion("polite", 0, dictionary, "timer");
+
+            expect(region.getAttribute("aria-live")).toBe("polite");
```

Comment:
**suggestion:** You could drop this assertion as it was already tested in a previous test, and instead only assert the part described in the test (which would be the role).

```suggestion

```

Reply:
The `aria-live` attribute is the part actually changing the politeness level, so I think I'll keep that in. I am refactoring it to be dynamic and less repetitive, though! Thanks for the suggestion.